ftw/referencewidget/behaviors.py

```diff
@@ -14,6 +14,12 @@
 from zope.interface import Interface
 
 
+if IS_PLONE_5_OR_GREATER:
+    from plone.app.dexterity import _
+else:
+    from plone.app.dexterity import MessageFactory as _
```

Comment:
I propose to use the standard Plone message factory: https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/__init__.py#L209
The domain is the same as in dexterity packages..

Reply:
👍 Good one, this avoids the `if`